### DISCUSSION TITLE
How configure fzf?

### DISCUSSION THREAD

**Discussion by knevan** [junegunn/fzf#4120] [Q&A] [answered]
I have a function like this in profile.ps1 for preview window:
```
$env:FZF_DEFAULT_OPTS = '--height 60% --layout reverse --border --preview-window right,60%,border-rounded,wrap'
```
But preview windows doesnt appear:
![image](https://github.com/user-attachments/assets/ac6c96b5-6fd2-4414-9772-efa609d3286b)

Its me trying to combine zoxide and fzf but failed:
```
function zcd {
    Invoke-Fzf -Query (zoxide query -l | Out-String) -Preview "pwsh.exe -Command Get-ChildItem -Path { $_ }" -PreviewWindow 'right:40%'
}
```
But got an error like this:
![image](https://github.com/user-attachments/assets/0fcc32f0-84ea-4d5e-8efd-7479135bd1cd)

**Comment by xiaoshihou514** [comment]
> But preview windows doesnt appear:

I believe you have to specify `--preview`
Example:
```shell
fzf --preview='bat --color=always -pp {1}'
```
> Its me trying to combine zoxide and fzf but failed:

`zoxide` has builtin fzf support, it's called `zi`. You have access to it if you have integrated zoxide with your shell (https://github.com/ajeetdsouza/zoxide?tab=readme-ov-file#installation)

**Reply by knevan** [reply]
>fzf --preview='bat --color=always -pp {1}'

Is it necessary to use bat or highlight to display the preview? Is it also possible without both?

**Reply by xiaoshihou514** [reply, marked as answer]
```shell
fzf --preview='cat {1}'
fzf --preview='head {1}'
```
